proj/src/views/Projects.js

```diff
@@ -2,13 +2,12 @@ import React, { Component } from 'react';
 import styled from 'styled-components';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import uuidv4 from 'uuid/v4';
 
 import EditProject from 'components/projects/EditProject';
 import withLoader from 'components/projects/withLoader'
 import ProjectList from 'components/projects/ProjectList'
 import ProjectPreview from 'components/projects/ProjectPreview';
-import { openEditProject } from 'actions/projectsActions'
+import { openEditProject, fetchProjects } from 'actions/projectsActions'
 
 import gridLayout from 'assets/images/grid.png'
 import listLayout from 'assets/images/list.png'
```

Comment:
btw why import these images and not put directly in the src attribute?

Reply:
Just looks cleaner